src/pages/CreateJobPost/CreateJobPost.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import CreateJobPostForm from "components/CreateJobPostForm/CreateJobPostForm";
+
+function CreateJobPost() {
+  return <CreateJobPostForm />;
+}
+
+export default CreateJobPost;
```

Comment:
Why do you need this component at all?

Reply:
Now there is only a form on the page, and most likely it will be until the end of the project, but the page must be changeable  and scalable. It's easy to add to this page whatever what client want. Job post form is separete component and it should be. So I believe that we need this page